src/css/cases/vosta.scss

```diff
@@ -0,0 +1,614 @@
+	background: url(#{$img}/waves.jpg) center top no-repeat;
+	@include background-size(cover);
+	transform-origin: 70% 0;
+}
+.mood-ship {
+	width: 499px;
+	height: 505px;
+	position: absolute;
+	bottom: 70%;
+	right: 3%;
+	background: url(#{$img}/ship.png) left bottom no-repeat;
+	transform-origin: -100% 100%;
+}
+
+/* case title */
+.case-summary {
+	min-width: 320px;
+	top: 40%;
+}
+.case-summary span {
+	color: #80bbe0;
+	font-style: normal;
+	text-transform: uppercase;
+}
+
+.arrow-wrap {
+	position: absolute;
+	bottom: 50px;
+	left: 50%;
+	.continue {
+		bottom: auto;
+		left: auto;
+	}
+}
+
+
+/* second client slide */
+
+.client-ship-back {
+	position: fixed;
+	top: 100%;
+	left: 0;
+	width: 162px;
+	height: 166px;
+	margin-top: 50%;
+	background: url(#{$img}/ship-back.png) 0 0 no-repeat;
+}
+.client-ship-front {
+	position: fixed;
+	top: 100%;
+	left: 0;
+	width: 396px;
+	height: 403px;
+	margin-top: 50%;
+	background: url(#{$img}/ship-front.png) 0 0 no-repeat;
+}
+.desc-client {
+	margin-top: 40vw;
+}
+/* all descriptions with orange dash */
+.description {
+	h2:after {
+		background: #f47318;
+	}
+}
+
+
+/* third slide with engine rotation */
+
+.grey-case {
+	position: fixed;
+	top: 100%;
+	width: 100%;
+	background: url(#{$img}/grey-gradient.jpg) center center no-repeat;
+	@include background-size(cover);
+}
+.grey-case:after {
+  	content: '';
+  	background: url(#{$img}/tab-360.png) 0 0 no-repeat;
+	width: 48px;
+	height: 77px;
+	display: block;
+	position: absolute;
+	bottom: -10px;
+	left: 50%;
+	margin-left: -24px;
+}
+.click-n-drag {
+	position: absolute;
+	left: 75%;
+	width: 100px;
+	top: 60px;
+}
+.click-n-drag:after {
+  	content: '';
+	position: absolute;
+	width: 45px;
+	height: 39px;
+	display: block;
+	position: absolute;
+	background: url(/img/cases/vosta/arrow.png) center center no-repeat;
+	margin: 0.8em 0 0 -45px;
+	left: 50%;
+}
+.spritespin-placeholder {
+	margin: 65px auto 140px;
+	padding-bottom: 50px;
+}
+.spritespin-placeholder:after {
+	content: '';
+  	background: url(#{$img}/engine-shadow.png) 0 0 no-repeat;
+	width: 548px;
+	height: 91px;
+	display: block;
+	position: absolute;
+	bottom: 40px;
+	left: 50%;
+	margin-left: -274px;
+}
+.desc-grey {
+	position: absolute;
+	margin-top: 100px;
+    width: 50%;
+    left: 25%;
+}
+
+
+/* slide with cards and sliding phone */
+
+.cards {
+	position: fixed;
+	top: 130%;
+	left: 0;
+	width: 100%;
+	height: 100%;
+	background: url(#{$img}/cards-wave.jpg) left top no-repeat #fff;
+}
+.cards-cards {
+	position: absolute;
+	width: 1110px;
+	height: 740px;
+	margin-top: -100px;
+	background: url(#{$img}/cards-cards.png) left top no-repeat;
+}
+.cards-phone {
+	position: absolute;
+	width: 761px;
+	height: 512px;
+	background: url(#{$img}/cards-phone.png) left top no-repeat;
+}
+.desc-cards {
+	position: absolute;
+	top: 60%;
+	left: 60%;
+	text-align: left;
+	h2 {
+		text-align: left;
+	}
+	h2:after {
+		margin-left: 0;
+	}
+}
+
+
+
+/* !!! done until here !!! */
```

Comment:
You can remove everything under this line, since it's for the kpmg case anyway. :)